@here/harp-mapview/lib/Utils.ts

```diff
@@ -242,53 +243,190 @@ export namespace MapViewUtils {
+
+        const headingAxis = projection.surfaceNormal(rotationTargetWorld, cache.vector3[0]);
+        const headingQuat = cache.quaternions[0].setFromAxisAngle(headingAxis, deltaAzimuth);
         camera.quaternion.premultiply(headingQuat);
         camera.position.sub(rotationTargetWorld);
         camera.position.applyQuaternion(headingQuat);
         camera.position.add(rotationTargetWorld);
+    }
+
+    /**
+     * @hidden
+     * @internal
+     *
+     * Clamps the supplied `deltaTilt` to the `maxTiltAngle` supplied. Note, when a non-zero offset
+     * is applied, we apply
+     */
+    function computeClampedDeltaTilt(
+        mapView: MapView,
+        offsetX: number,
+        offsetY: number,
+        deltaTilt: number,
+        maxTiltAngle: number,
+        rotationTargetWorld: THREE.Vector3
+    ): number {
+        const camera = mapView.camera;
+        const projection = mapView.projection;
 
         const mapTargetWorld =
             offsetX === 0 && offsetY === 0
                 ? rotationTargetWorld
                 : MapViewUtils.rayCastWorldCoordinates(mapView, 0, 0);
         if (mapTargetWorld === null) {
-            return;
+            return 0;
         }
-        const mapTargetNormal = projection.surfaceNormal(mapTargetWorld, new THREE.Vector3());
 
-        camera.updateMatrixWorld(true);
-        const dir = cache.vector3[1].setFromMatrixColumn(camera.matrixWorld, 2);
-        const up = cache.vector3[2].setFromMatrixColumn(camera.matrixWorld, 1);
+        let currentTilt = 0;
+        {
+            const mapTargetNormal = projection.surfaceNormal(mapTargetWorld, cache.vector3[0]);
+            camera.updateMatrixWorld(true);
 
-        // Due to inaccuracies it can happen that the tilt angle gets less than 0.
-        // Using the dot product of the direction vector and the surface normal alone does not give us the sign.
-        const tiltSign = Math.sign(up.dot(mapTargetNormal));
+            const dir = cache.vector3[1].setFromMatrixColumn(camera.matrixWorld, 2);
+            const up = cache.vector3[2].setFromMatrixColumn(camera.matrixWorld, 1);
 
-        const currentTilt = (tiltSign === 0 ? 1 : tiltSign) * dir.angleTo(mapTargetNormal);
+            // Due to inaccuracies it can happen that the tilt angle gets less than 0.
+            // Using the dot product of the direction vector and the surface normal alone does not give us the sign.
+            const tiltSign = Math.sign(up.dot(mapTargetNormal));
+            currentTilt = (tiltSign === 0 ? 1 : tiltSign) * dir.angleTo(mapTargetNormal);
+        }
 
-        //FIXME(HARP-11926): For globe tilt in the map center is different from the tilt in the rotation center,
-        //hence the clamped tilt is too conservative.
+        const currentTiltRotationCenter = MapViewUtils.extractTiltAngleFromLocation(
+            mapView.projection,
+            mapView.camera,
+            rotationTargetWorld!
+        );
```

Comment:
we should skip doing this if offset is 0 (when mapTarget === rotationTarget). In this case currentTiltRotationCenter === currentTilt